scripts/toolchain/requirements.txt

```diff
@@ -7,3 +7,8 @@ GitPython==3.1.37
 pyserial==3.5
 ansi==0.3.6
 kconfiglib==14.1.0
+mido==1.3
+python-rtmidi
+iterfzf
+psutil
+tqdm
```

Comment:
Should lock versions of these dependencies. It might also be wise to make sure this PR branch doesn't break the CI image build. There's a script to simulate what the GH runner will do, here: https://github.com/SynthstromAudible/dbt-toolchain/blob/218b2352b3449045c620de6e306435c372c79ddc/actions-support/simulate_ci_image_build.sh and you'll need to hack the docker manifest here: https://github.com/SynthstromAudible/dbt-toolchain/blob/218b2352b3449045c620de6e306435c372c79ddc/actions-support/ci-image/Dockerfile#L48-L49.

This will require a few gigs of storage space for all the toolchains, and might take 30 minutes or so to run. I'll try to run this myself in the next few days or so if you can't get to it.